### DISCUSSION TITLE
ilogtail collect logdata exception,help

### DISCUSSION THREAD

**Discussion by yecun123** [alibaba/ilogtail#651] [Help] [answered]
描述： ilogtail 正常采集日志过程中，我们终止掉 ilogtail进程，过一段时间我们再重启ilogtail 进程，这期间会出现问题。
如日志出现了分割（按日志大小或时间间隔），重启ilogtail后，发现采集上来的数据不是连续的，分割后的文件只采集后部分，前部分日志采集丢掉了
1. ilogtail 配置
![image](https://user-images.githubusercontent.com/125344638/218667059-4fc92597-d7fe-4075-97c4-711ff1beb67b.png)
2. 日志采集结果
① 14：06启动ilogtail
![image](https://user-images.githubusercontent.com/125344638/218665864-3744b872-ef42-4401-b236-f404b54477ee.png)
② 采集匹配的日志出现分割
![image](https://user-images.githubusercontent.com/125344638/218666001-475b802d-5c28-4623-aa0a-1703fc127bd7.png)
③ 采集上来的数据结果
![image](https://user-images.githubusercontent.com/125344638/218666515-20f86745-7c1e-4373-bd43-c5b8fb519fb2.png)

**Comment by linrunqi08** [comment]
请问ilogtail停止了多久？然后再重启呢

**Reply by yecun123** [reply]
一般有 5 min以上，请问这个有什么限制吗？ 停止久了ilogtail 会重新采集？

**Comment by linrunqi08** [comment, marked as answer]
可以配置下polling_ignore_file_modify_timeout和polling_file_first_watch_timeout这两个参数试下

**Reply by yecun123** [reply]
这是启动参数 ？放在 ilogtail_config.json里是吧？官方文档上没看到这个 ，单位是毫秒还是秒呢

**Reply by linrunqi08** [reply]
单位是秒

**Reply by yecun123** [reply]
已修复，官方文档未找到该两字段含义，心中有部分猜测，麻烦大佬告知下两个参数的具体意义
